### PR TITLE
[SSL + DNS] Various updates

### PR DESCRIPTION
**Pull request by kodster28**
- PCX-4675 (pending questions)
- PCX-4295
- PCX-4660 (partial)

### CODE REVIEW

File: content/ssl/ssl-tls/cipher-suites.md [hunk 1]
```diff
@@ -16,33 +17,46 @@ We no longer support [RC4 cipher suites](https://blog.cloudflare.com/end-of-the-
...
+| ECDHE-ECDSA-AES256-SHA384           | TLS 1.2 | Compatible | [0xc024] | TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384 |
+| ECDHE-RSA-AES256-GCM-SHA384         | TLS 1.2 | Modern | [0xc030] | TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384 |
+| ECDHE-RSA-AES256-SHA384             | TLS 1.2 | Compatible | [0xc028] | TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384 |
+| ECDHE-RSA-AES256-SHA                | TLS 1.0 | Compatible | [0xc014] | TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA |
+| AES256-GCM-SHA384                   | TLS 1.2 | Legacy | [0x9d] | TLS_RSA_WITH_AES_256_GCM_SHA384 |
+| AES256-SHA256                       | TLS 1.2 | Legacy | [0x3d] | TLS_RSA_WITH_AES_256_CBC_SHA256
+ |
+| AES256-SHA                          | TLS 1.0 | Legacy | [0x35] | TLS_RSA_WITH_AES_256_CBC_SHA |
+| DES-CBC3-SHA                        | TLS 1.0 | Legacy | [0x0701c0] | SSL_CK_DES_192_EDE3_CBC_WITH_SHA |
+| AEAD_AES_128_GCM_SHA256 [^1]        | TLS 1.3 | Modern | N/A | N/A |
```
Comment (maxnystrom) [inline content/ssl/ssl-tls/cipher-suites.md]: Here, I would update the OpenSSL and IANA names for TLS 1.3 ciphers to be identical to what is seen here https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#tls-parameters-4

Which is essentially what is also documented here https://www.rfc-editor.org/rfc/rfc8446.html#appendix-B.4

Comment (kodster28) [reply content/ssl/ssl-tls/cipher-suites.md]: As in, both the IANA and OpenSSL names should be the same (`TLS_AES_128_GCM_SHA256`)?

Or we should keep the OpenSSL name as `AEAD_AES_128_GCM_SHA256` and then list the corresponding IANA name `TLS_AES_128_GCM_SHA256`?

**Review by maxnystrom — APPROVED** [review]

File: content/dns/zone-setups/zone-transfers/cloudflare-as-primary/setup.md [hunk 2]
```diff
@@ -12,27 +12,27 @@ With [outgoing zone transfers](/dns/zone-setups/zone-transfers/cloudflare-as-pri
...
 
-When Cloudflare performs outgoing transfers, we also transfer out the IP or hostname of each DNS record. This means that - if Cloudflare (as primary) and your secondary provider are both authoritative - they will not reply with the same response for proxied DNS records. Cloudflare would respond with two [Cloudflare edge IP addresses](https://www.cloudflare.com/ips) and your secondary provider would respond with the origin IP or hostname.
+When Cloudflare performs outgoing transfers, we transfer out the origin IP or hostname of each DNS record. This means that - if Cloudflare (as primary) and your secondary provider are both authoritative - they will not reply with the same response for proxied DNS records. Cloudflare would respond with two [Cloudflare edge IP addresses](https://www.cloudflare.com/ips) and your secondary provider would respond with the origin IP or hostname.
 
 {{<Aside type="note">}}
 
-Cloudflare is actively working to support outgoing zone transfers of proxied records.
+If you need Cloudflare to transfer out edge IP addresses instead of origin IP addresses or hostname for proxied records, reach out to your account team for further instructions.
```
Comment (crwaters16) [inline content/dns/zone-setups/zone-transfers/cloudflare-as-primary/setup.md]: hostnames (plural)?

File: content/ssl/ssl-tls/cipher-suites.md [hunk 3]
```diff
@@ -16,35 +17,47 @@ We no longer support [RC4 cipher suites](https://blog.cloudflare.com/end-of-the-
...
+| AES256-SHA256                       | TLS 1.2 | Legacy | [0x3d] | TLS_RSA_WITH_AES_256_CBC_SHA256 |
+| AES256-SHA                          | TLS 1.0 | Legacy | [0x35] | TLS_RSA_WITH_AES_256_CBC_SHA |
+| DES-CBC3-SHA                        | TLS 1.0 | Legacy | [0x0701c0] | SSL_CK_DES_192_EDE3_CBC_WITH_SHA |
+| TLS_AES_128_GCM_SHA256[^1]        | TLS 1.3 | Modern | {0x13,0x01} | TLS_AES_128_GCM_SHA256 |
+| TLS_AES_256_GCM_SHA384[^1]        | TLS 1.3 | Modern | {0x13,0x02} | TLS_AES_256_GCM_SHA384 |
+| TLS_CHACHA20_POLY1305_SHA256[^1]  | TLS 1.3 | Modern | {0x13,0x03} | TLS_CHACHA20_POLY1305_SHA256 |
+
+[^1]: Automatically supported by your zone if you [enable TLS 1.3](/ssl/edge-certificates/additional-options/tls-13/#enable-tls-13). TLS 1.3 uses the same cipher suite space as previous versions of TLS, but define these cipher suites differently. They only specify the symmetric ciphers and cannot be used for TLS 1.2. Similarly, TLS 1.2 and lower cipher suites cannot be used with TLS 1.3 (IETF TLS 1.3 draft 21). BoringSSL also hard-codes cipher preferences in this order for TLS 1.3.
```
Comment (crwaters16) [inline content/ssl/ssl-tls/cipher-suites.md]:
```suggestion
[^1]: Automatically supported by your zone if you [enable TLS 1.3](/ssl/edge-certificates/additional-options/tls-13/#enable-tls-13). TLS 1.3 uses the same cipher suite space as previous versions of TLS, but defines these cipher suites differently. TLS 1.3 only specifies the symmetric ciphers and cannot be used for TLS 1.2. Similarly, TLS 1.2 and lower cipher suites cannot be used with TLS 1.3 (IETF TLS 1.3 draft 21). BoringSSL also hard-codes cipher preferences in this order for TLS 1.3.
```

Comment (crwaters16) [reply content/ssl/ssl-tls/cipher-suites.md]: Uncertain which thing "they" referred to, correct if it's not TLS 1.3

**Review by crwaters16 — APPROVED** [review]

**Pull request merged**